src/components/react/SortTable.tsx

```diff
@@ -36,8 +37,7 @@ function SortTable({ projectData }: { projectData: ProjectData[] }) {
   const [loading, setLoading] = useState(true);
   const [languageFilter, setLanguageFilter] = useState<string | null>(null);
 
-useEffect(() => {
-  (async () => {
+  const debouncedGitHubApiCall = debounce(async () => {
```

Comment:
sorry, maybe i dont fully understand the code, but i think u should use here useCallback otherwise the debounced function will be re-created on each re-render

Reply:
I don't think it is a problem in our project.